### ISSUE TITLE
Capture information for proposals transactions

### ISSUE THREAD

**Issue by vishakh** [Cryptonomic/Conseil#453]
A typical proposals operation looks like this when returned by the Tezos RPC API:

```json
{
            "protocol": "PsddFKi32cMJ2qPjf43Qv5GDWLDPZb3T3bF6fLKiF5HtvHNU7aP",
            "chain_id": "NetXdQprcVkpaWU",
            "hash": "opS9cGJ51jCcLwkofdnQbwLnKNoqBzuQMfoMXPviS99kszQHiRQ",
            "branch": "BMchFBNHU3BA8AtAVCwyRA9SpWNSrbtzA8K6WNUNpVxxNcApHqh",
            "contents": [
                {
                    "kind": "proposals",
                    "source": "tz1MecudVJnFZN5FSrriu8ULz2d6dDTR7KaM",
                    "period": 10,
                    "proposals": [
                        "Psd1ynUBhMZAeajwcZJAeq5NrxorM6UCU4GJqxZ7Bx2e9vUWB6z"
                    ],
                    "metadata": {}
                }
            ],
            "signature": "sigoEwr6zeS28sufRgBznqAjf2jnwPxYw3vG3L5xVSYdJgJoACxVU9n4BrUjU8QJNdB9w7c4gxLYmaFzVJgdRD4gB4HroviY"
        }
```

We are currently not capturing any of these fields. Lorre should start picking them up.

**Comment by anonymoussprocket** [comment]
Proposal is an operation and should be stored this way. We already have code to deal with operations and this should be added there. Store the `proposals` array as a single field.

Completion of this ticket should obviate the need for the separate proposal table. @bhaskarkishore, can you opine?

**Comment by anonymoussprocket** [comment]
related to #520

**Comment by ivanopagano** [comment]
Closed by #536 ?

@anonymoussprocket @ba205

**Comment by ba205** [comment]
This should be closed, assuming master in dev is functioning just fine. I'm not sure about deleting the table yet though, as proposals is a one-to-many relation, and I'm not sure if the stringified array representation in master is best. @anonymoussprocket, thoughts?
